collect_app/src/main/res/values/strings.xml

```diff
@@ -400,4 +400,13 @@
 <string name="geotrace_view_change_location">View or Change GeoTrace</string>
 <string name="bearing">Bearing: %.3f</string>
 <string name="direction">Direction: %s</string>
+<string name="reset_settings">Settings</string>
+<string name="reset_filled_out_forms">Filled out forms</string>
+<string name="reset_dialog_nothing">You have not selected anything to reset.</string>
+<string name="reset_dialog_finished">Successfully reset app state. Tap OK to close the app.</string>
```

Comment:
@lognaturel This is the only piece of copy I haven't updated, mostly because at this point we are just closing the app and users will have to start it themselves (I tried to implement relaunching but it was not working as expected and was not very reliable). Lmk if I should still update the copy.